mbf_utility/include/mbf_utility/robot_information.h

```diff
@@ -69,7 +71,20 @@ class RobotInformation
    */
   bool getRobotPose(geometry_msgs::PoseStamped &robot_pose) const;
 
-  bool getRobotVelocity(geometry_msgs::TwistStamped &robot_velocity, ros::Duration look_back_duration) const;
+  /**
+   * @brief Returns the current robot velocity, as provided by the odometry helper.
+   * @param robot_velocity Reference to the robot_velocity message object to be filled.
+   * @return true, if the current robot velocity could be obtained, false otherwise.
+   */
+  bool getRobotVelocity(geometry_msgs::TwistStamped &robot_velocity) const;
+
+  /**
+   * @brief Check whether the robot is stopped or not
+   * @param rot_stopped_velocity The rotational velocity below which the robot is considered stopped
+   * @param trans_stopped_velocity The translational velocity below which the robot is considered stopped
+   * @return true if the robot is stopped, false otherwise
+   */
+  bool isRobotStopped(double rot_stopped_velocity, double trans_stopped_velocity) const;
```

Comment:
this is unused, is that intentional?

Reply:
Yes; we want to replace base_local_planner with the same functionality, and it provides this check